packages/cra-template-mephisto-review/template/src/ItemRenderer.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { useMephistoReview } from "mephisto-review-hook";
+import { useParams, useHistory } from "react-router-dom";
+import { Button, H2, Card, Elevation } from "@blueprintjs/core";
+import AppToaster from "./components/AppToaster";
+import "./ItemRenderer.css";
+
+function App() {
+  const { id } = useParams();
+  const {
+    data,
+    isFinished,
+    isLoading,
+    submit,
+    error,
+    mode,
+  } = useMephistoReview({ taskId: id });
+  const history = useHistory();
+
+  const confirmReview = () => {
+    if (mode === "OBO") {
+      history.push("/");
+    } else {
+      AppToaster.show({ message: "Review Sent!" });
+    }
+  };
+
+  const reviewError = (error) => {
+    AppToaster.show({ message: `ERROR: ${error}` });
+  };
+
+  return (
+    <main className="Item-Renderer">
+      {mode === "ALL" ? (
+        <Button
+          onClick={() => history.push("/")}
```

Comment:
I think this can also be wrapped in a Link tag. Generally, using Link tags are better for browser accessibility. You also get to see the destination URL when hovering over the tag